packages/build/bin/utils.js

```diff
@@ -56,9 +56,14 @@ function getConfigFile(name, defaultName) {
 
 /**
  * Resolve the path of the cli command
- * @param {string} cli
+ * @param {string} cli CLI module path
+ * @param {boolean} resolveFromProjectFirst Try to resolve the CLI path from the
+ * package dependencies of the current project first instead of `@loopback/build`
  */
-function resolveCLI(cli) {
+function resolveCLI(cli, resolveFromProjectFirst = true) {
```

Comment:
Please don't use boolean arguments, they lead to code that's difficult to read. Use an `options` object with named properties.

```suggestion
function resolveCLI(cli, {resolveFromProjectFirst} = {resolveFromProjectFirst: true}) {
```